[bug] CMake errors on develop branch while compiling

## Bug while compiling

Error on `lgsvl_simulator_bridge`:
```
Errors     << lgsvl_simulator_bridge:install /root/catkin_ws/logs/lgsvl_simulator_bridge/build.install.000.log
CMake Error at cmake_install.cmake:133 (file):
  file INSTALL cannot find
  "/root/catkin_ws/src/ros/src/simulation/lgsvl_simulator_bridge/include".
```

Error on `autoware_driveworks_gmsl_interface`
```
Errors     << autoware_driveworks_gmsl_interface:install /root/catkin_ws/logs/autoware_driveworks_gmsl_interface/build.install.000.log
make: *** No rule to make target 'install'.  Stop.
```

### Specifications

  - Ubuntu version: 16.04
  - ROS version: Kinetic
  - Autoware branch: develop

--- a/ros/src/driveworks/packages/autoware_driveworks_gmsl_interface/CMakeLists.txt
+++ b/ros/src/driveworks/packages/autoware_driveworks_gmsl_interface/CMakeLists.txt
@@ -3,16 +3,7 @@ project(autoware_driveworks_gmsl_interface)
 
 find_package(CUDA)
 
-if ("${CMAKE_SYSTEM_PROCESSOR}" STREQUAL "aarch64" AND CUDA_FOUND)
-    if(CMAKE_CXX_COMPILER_ID STREQUAL "GNU")
-      set(DRIVEWORKS_LINKER_FLAGS "-Wl,--unresolved-symbols=ignore-in-object-files -Wl,--allow-shlib-undefined")
-    elseif(CMAKE_CXX_COMPILER_ID MATCHES "Clang")
-      set(DRIVEWORKS_LINKER_FLAGS "-Wl,-undefined,dynamic_lookup")
-    endif()
-
-    add_definitions(-DVIBRANTE)
-
-    find_package(catkin REQUIRED COMPONENTS
+find_package(catkin REQUIRED COMPONENTS
         autoware_build_flags
         roscpp
         sensor_msgs
@@ -21,14 +12,23 @@ if ("${CMAKE_SYSTEM_PROCESSOR}" STREQUAL "aarch64" AND CUDA_FOUND)
         image_transport
         camera_info_manager
         nodelet
-    )
+        )
 
-    catkin_package(
+catkin_package(
         LIBRARIES ${PROJECT_NAME}
         CATKIN_DEPENDS
         sensor_msgs
         std_msgs
-    )
+)
+
+if ("${CMAKE_SYSTEM_PROCESSOR}" STREQUAL "aarch64" AND CUDA_FOUND)
+    if(CMAKE_CXX_COMPILER_ID STREQUAL "GNU")
+      set(DRIVEWORKS_LINKER_FLAGS "-Wl,--unresolved-symbols=ignore-in-object-files -Wl,--allow-shlib-undefined")
+    elseif(CMAKE_CXX_COMPILER_ID MATCHES "Clang")
+      set(DRIVEWORKS_LINKER_FLAGS "-Wl,-undefined,dynamic_lookup")
+    endif()
+
+    add_definitions(-DVIBRANTE)
 
     include_directories(
         include

--- a/ros/src/driveworks/packages/autoware_driveworks_gmsl_interface/CMakeLists.txt
+++ b/ros/src/driveworks/packages/autoware_driveworks_gmsl_interface/CMakeLists.txt
@@ -3,30 +3,30 @@ project(autoware_driveworks_gmsl_interface)
 
 find_package(CUDA)
 
-find_package(catkin REQUIRED COMPONENTS
-        autoware_build_flags
-        roscpp
-        sensor_msgs
-        std_msgs
-        cv_bridge
-        image_transport
-        camera_info_manager
-        nodelet
-        )
-
-catkin_package(
-        LIBRARIES ${PROJECT_NAME}
-        CATKIN_DEPENDS
-        sensor_msgs
-        std_msgs
-)
-
 if ("${CMAKE_SYSTEM_PROCESSOR}" STREQUAL "aarch64" AND CUDA_FOUND)
     if(CMAKE_CXX_COMPILER_ID STREQUAL "GNU")
       set(DRIVEWORKS_LINKER_FLAGS "-Wl,--unresolved-symbols=ignore-in-object-files -Wl,--allow-shlib-undefined")
     elseif(CMAKE_CXX_COMPILER_ID MATCHES "Clang")
       set(DRIVEWORKS_LINKER_FLAGS "-Wl,-undefined,dynamic_lookup")
     endif()
+
+    find_package(catkin REQUIRED COMPONENTS
+            autoware_build_flags
+            roscpp
+            sensor_msgs
+            std_msgs
+            cv_bridge
+            image_transport
+            camera_info_manager
+            nodelet
+    )
+
+    catkin_package(
+            LIBRARIES ${PROJECT_NAME}
+            CATKIN_DEPENDS
+            sensor_msgs
+            std_msgs
+    )
 
     add_definitions(-DVIBRANTE)
 
@@ -91,5 +91,7 @@ if ("${CMAKE_SYSTEM_PROCESSOR}" STREQUAL "aarch64" AND CUDA_FOUND)
     )
 
 else()
+    find_package(catkin REQUIRED)
+    catkin_package()
     message("gmsl_camera driver will not be built. CUDA was not found.")
 endif()

--- a/ros/src/driveworks/packages/autoware_driveworks_gmsl_interface/package.xml
+++ b/ros/src/driveworks/packages/autoware_driveworks_gmsl_interface/package.xml
@@ -10,7 +10,7 @@
   <license>BSD</license>
 
   <buildtool_depend>catkin</buildtool_depend>
-  <!-- <buildtool_depend>autoware_build_flags</buildtool_depend> -->
+  <buildtool_depend>autoware_build_flags</buildtool_depend>
   <build_depend>libglew-dev</build_depend>
   <build_depend>roscpp</build_depend>
   <build_depend>sensor_msgs</build_depend>

--- a/ros/src/simulation/lgsvl_simulator_bridge/CMakeLists.txt
+++ b/ros/src/simulation/lgsvl_simulator_bridge/CMakeLists.txt
@@ -1,61 +1,42 @@
 cmake_minimum_required(VERSION 2.8.3)
 project(lgsvl_simulator_bridge)
 
-## Compile as C++11, supported in ROS Kinetic and newer
 add_compile_options(-std=c++11)
 
 find_package(catkin REQUIRED COMPONENTS
-  autoware_msgs
-  geometry_msgs
-  roscpp
-  rospy
-  roslib
-  sensor_msgs
-)
+        autoware_msgs
+        geometry_msgs
+        roscpp
+        rospy
+        roslib
+        sensor_msgs
+        )
 
-###################################
-## catkin specific configuration ##
-###################################
-## The catkin_package macro generates cmake config files for your package
-## Declare things to be passed to dependent projects
-## INCLUDE_DIRS: uncomment this if your package contains header files
-## LIBRARIES: libraries you create in this project that dependent projects also need
-## CATKIN_DEPENDS: catkin_packages dependent projects also need
-## DEPENDS: system dependencies of this project that dependent projects also need
 catkin_package(
-#  INCLUDE_DIRS include
-#  LIBRARIES lgsvl_simulator_bridge
-  CATKIN_DEPENDS autoware_msgs geometry_msgs roscpp rospy sensor_msgs
-#  DEPENDS system_lib
+        CATKIN_DEPENDS
+        autoware_msgs
+        geometry_msgs
+        roscpp
+        rospy
+        sensor_msgs
 )
 
-###########
-## Build ##
-###########
 
-## Specify additional locations of header files
-## Your package locations should be listed before other locations
 include_directories(
-  include
-  ${catkin_INCLUDE_DIRS}
+        ${catkin_INCLUDE_DIRS}
 )
 
-#############
-## Install ##
-#############
-foreach(programs)
-  install(TARGETS ${programs}
-    RUNTIME DESTINATION ${CATKIN_PACKAGE_BIN_DESTINATION}
-)
-endforeach(nodes)
+foreach (programs)
+    install(TARGETS ${programs}
+            RUNTIME DESTINATION ${CATKIN_PACKAGE_BIN_DESTINATION}
+            )
+endforeach (programs)
 
-install(DIRECTORY include
-  DESTINATION ${CATKIN_PACKAGE_INCLUDE_DESTINATION}
-)
+foreach (dir launch rviz)
+    install(DIRECTORY ${dir}/
+            DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION}/${dir})
+endforeach (dir)
 
-foreach(dir launch rviz)
-  install(DIRECTORY ${dir}/
-    DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION}/${dir})
-endforeach(dir)
-
-install(PROGRAMS scripts/launch_sim.bash DESTINATION ${CATKIN_PACKAGE_BIN_DESTINATION})
+install(PROGRAMS scripts/launch_sim.bash
+        DESTINATION ${CATKIN_PACKAGE_BIN_DESTINATION}
+        )

